_data/accessibility-tests/breadcrumb.yml

```diff
@@ -0,0 +1,125 @@
+  - summary: Breadcrumb separators (arrows) meet color contrast requirements.    
+    summary_additional: |
+      When you use a color contrast checker to view the breadcrumb navigation, 
+      contrast between the separators (arrows) and background colors is at least 3:1. 
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.11
+  - summary: Breadcrumb links appear in the same location and order on every page of the website.
+    summary_additional: |
+      When you navigate to different pages on the website, the breadcrumb links are 
+      always in the same order and in the same place.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.2.3
+  - summary: Breadcrumb links provide location within a set of pages. 
+    summary_additional: |
+      When navigating through different pages, 
+      the breadcrumb provides clarity on your location on the site.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.8
+  - summary: Color is not the only method used to convey the breadcrumb link states. 
+    summary_additional: |
+      When navigating through the breadcrumb links, you can read text to understand what page you are on. 
+      There may also be underlining missing to indicate you are on the current page.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.1
+# Zoom/screen magnification tests
+  - summary: Breadcrumb text and links are visible and functional when screen is magnified. 
+    summary_additional: |
+      When you zoom in to 200%, breadcrumb link text is still readable and legible and properly sized.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.4
+  - summary: Breadcrumb text and links can wrap when magnified more than 200%. 
+    summary_additional: |
+      When you zoom in to 400%, longer breadcrumb link text may wrap, 
+      but is still readable and legible and properly sized.
+    test_status: conditional
+    test_type: zoom
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.10 
+# Keyboard navigation tests
+  - summary: Keyboard can access all links in the breadcrumb. 
+    summary_additional: |
+      When you navigate through the breadcrumb navigation with a keyboard, 
+      you can interact with every link inside it.
+    status_details: | 
+      Breadcrumb component may not be fully interactive with a keyboard. 
+      The wording for the current page in the breadcrumb list may not be focusable with a keyboard 
+      nor recognizable as the current page using a screen reader. You will need to 
+      test on your own implementation. More information: <a href='https://github.com/uswds/uswds-site/issues/2818'>USWDS #2818</a>. 
+    github_issue_number: 6054
+    test_status: fail
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.1 
```

Comment:
**todo:** Update test_status to pass

After discussion, we decided that in it's current state this component **does** pass this test.

Additional enhancement to update the `span` be wrapped in an anchor tag is being tracked in uswds/uswds#6054

```suggestion
  - summary: Keyboard can access all links in the breadcrumb. 
    summary_additional: |
      When you navigate through the breadcrumb navigation with a keyboard, 
      you can interact with every link inside it.
    github_issue_number: 6054
    test_status: pass
    test_type: keyboard
    version_tested: 3.8.2
    wcag_criterion: 2.1.1 
```